lib/transport/tests/test_aux_data.c

```diff
@@ -117,8 +116,7 @@ test_aux_data_added_nvpairs_are_returned_by_foreach_in_order(void)
   free_aux(aux);
 }
 
-static void
-test_aux_data_copy_creates_an_identical_copy(void)
+Test(aux_data, test_aux_data_copy_creates_an_identical_copy)
 {
   LogTransportAuxData *aux = construct_aux_with_some_data();
```

Comment:
```
LogTransportAuxData *aux = construct_aux_with_some_data();
[...]
free_aux(aux);
```
are repetitive pattern. Aux could be constructed in the `TestSuite` setup function, free could be called in the teardown function.

Reply:
Yes, I will change it.

Reply:
Yes, that is a good suggestion.